frameworks/Python/API-Hour/hello/hello/servers/yocto_http.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+
+import ujson
+
+from ..utils import generate_http_response
+
+class YoctoHttpJson(asyncio.Protocol):
+    def connection_made(self, transport):
+        self.transport = transport
+
+    def data_received(self, data):
+        # self.transport.write(data)
+        payload = ujson.dumps({'message': 'Hello, World!'})
+        self.transport.write(generate_http_response(payload))
+
+class YoctoHttpText(asyncio.Protocol):
+    def connection_made(self, transport):
+        self.transport = transport
+
+    def data_received(self, data):
+        # self.transport.write(data)
+        payload = 'Hello, World!'
+        self.transport.write(generate_http_response(payload, 'text/plain; charset=UTF-8'))
```

Comment:
plaintext test uses pipelined request.
Skipping HTTP may not work.

I feel skipping HTTP is less than "stripped". "stripped" means skip some default middleware in framework.